defaults/main.yml

```diff
@@ -17,8 +17,9 @@ dell_dsu_prerequisites:
   - usbutils
   - compat-libstdc++-33.i686
   - OpenIPMI
-dell_dsu_exit_codes:
+dell_dsu_changed_exit_codes:
   - 0
```

Comment:
Hi!

Could you document in this PR or even in the defaults file what these different exit codes mean? Perhaps with that link?

Reply:
Sure, I'll create another commit; just squash them on merge

Reply:
Thanks!